src/views/resources/ResourceListView.js

```diff
@@ -113,18 +113,19 @@ export const ResourceListView = () => {
     }
 
     const handleRescNameChange = (event) => {
-        setRescName(event.target.value);
+        // trim all whitespace in the resource name
+        setRescName(event.target.value.replace(/\s/g, ""));
```

Comment:
not convinced we need to remove *all* spaces (rather than just `trim()`)...

also, this is not all whitespace... could be a tab character or line feed...

Reply:
right, it's kinda wip and a little bit of back and forth. I think we can use trim() for those three fields (name, hostname, vault path), will fix that.

Reply:
agreed.